.github/workflows/kata-deploy.yaml

```diff
@@ -0,0 +1,34 @@
+on: push
+
+name: Build, Test, and Publish kata-deploy
+
+jobs:
+  kata-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: build-and-push-kata-deploy-ci
+        run: |
```

Comment:
Do these shell blocks run with `set -e` by default? If not, can we force that somehow?